pkg/pulumiyaml/run.go

```diff
@@ -685,6 +685,37 @@ func (r *Runner) Evaluate(ctx *pulumi.Context) syntax.Diagnostics {
 	return r.Run(programEvaluator{evalContext: eCtx, pulumiCtx: ctx})
 }
 
+func (r *Runner) getPulumiConfNodes() []configNode {
+	var projConf map[string]interface{}
+	var projConfStr = os.Getenv("PULUMI_CONFIG")
+	nodes := make([]configNode, 0)
+
+	if err := json.Unmarshal([]byte(projConfStr), &projConf); err != nil {
```

Comment:
We shouldn't swallow errors here. There are many reasons that `err != nil` beyond `projConfStr` being empty. We can actually check if it is empty; see above.